rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1078,6 +1078,7 @@ private void processPegoutsIndividually(
             .stream()
             .anyMatch(entry -> walletBalance.isGreaterThan(entry.getAmount()) || walletBalance.equals(entry.getAmount()));
         if (!canProcessAtLeastOnePegout) {
+            logger.warn("[processPegoutsIndividually] wallet balance {} cannot process at least one pegout", walletBalance);
```

Comment:
```suggestion
            logger.warn("[processPegoutsIndividually] wallet balance {} cannot process even one pegout", walletBalance);
```

Reply:
I Think At **Least One** Is Clearly..
For Example, We Have 5 Requests, A Check Is Done To Make Sure It Can Process At Least One Of Them Else It Logs That Warning And Returns. Pending When We Have Enough Funds